modules/twinklefluff.js

```diff
@@ -118,10 +129,14 @@ Twinkle.fluff.linkBuilder = {
 			Twinkle.fluff.disableLinks(revNode);
 		});
 
-		vandNode.appendChild(vandLink);
-		normNode.appendChild(normLink);
+		normNode.setAttribute('class', 'tw-rollback-link-normal');
+		vandNode.setAttribute('class', 'tw-rollback-link-vandalism');
 
-		var separator = inline ? ' ' : ' || ';
+		normNode.appendChild(sepNode1);
+		vandNode.appendChild(sepNode2);
+
+		normNode.appendChild(normLink);
+		vandNode.appendChild(vandLink);
 
 		if (!inline) {
 			var agfNode = document.createElement('strong');
```

Comment:
You have changed norm and vand buttons from `<strong>` to `<span>`, but you forget it for agf button.

Reply:
That's correct, sorry. Thanks for catching this.

Reply:
Fixed in 9992f5d1a10835b636a853bd5692b94f713a72f5:

![twinklefluff-20210430-patchfix1](https://user-images.githubusercontent.com/66693882/116628887-5ac0e180-a950-11eb-9376-8d6b69335deb.png)

![twinklefluff-20210430-patchfix2](https://user-images.githubusercontent.com/66693882/116628888-5b597800-a950-11eb-83f2-245bf52222d9.png)

![twinklefluff-20210430-patchfix3](https://user-images.githubusercontent.com/66693882/116628889-5b597800-a950-11eb-8b6c-f907c49129ad.png)